### PR TITLE
Update getting started notebook with improved examples

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

**Review by jpekmez — DISMISSED** [review]

File: cohere_sagemaker/client.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def generate(
         # not implemented in API
         # stop_sequences: List[str] = None,
         # this changed from 'NONE'
-        return_likelihoods: str = None,
+        return_likelihoods: str = 'NONE',
```
Comment (daulet) [inline cohere_sagemaker/client.py]: i dont think that works

Comment (CoderHam) [reply cohere_sagemaker/client.py]: Fixed

**Review by co-sudip — DISMISSED** [review]

File: example/sample_input.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "prompt": "Tell me about the last",
+    "max_tokens": 20,
+    "temperature": 0.75,
+    "k": 0,
+    "p": 0.75,
+    "frequency_penalty": 0,
+    "presence_penalty": 0,
+    "return_likelihoods": "NONE"
```
Comment (daulet) [inline example/sample_input.json]: is this right?

**Review by daulet — DISMISSED** [review]

**Review by daulet — DISMISSED** [review]

File: example/sample_input.json (hunk 2 above)
Comment (CoderHam) [reply example/sample_input.json]: Fixed to remove return_likelihoods since it is empty if set to None

**Review by daulet — APPROVED** [review]

**Pull request merged**